src/components/app/App.jsx

```diff
@@ -49,6 +50,15 @@ const App = () => {
                 </AuthenticatedPage>
               )}
             />
+            <PageRoute
+              exact
+              path="/:enterpriseSlug/executive-education-2u-enrollment-completed"
```

Comment:
I think a better path would be
`/:enterpriseSlug/executive-education-2u/enrollment-completed`

Reply:
@saleem-latif I've updated the URL. Please make the related changes in your PR.